index.html

```diff
@@ -46,9 +46,10 @@
 <div class="div_carlo">
 <img class="carlo_mob class_disable" src="img/carlo.gif" alt="image">
 <div class="carlo_txt animated" style="display:none;">
-<h2>Some 10 years ago, on a sultry April morning, a bright young man found himself standing outside a pretty purple villa at Dona Paula. </h2>
-<p>It was Ravi Shankar’s first day at Monte Carlo, home to Synapse and its many PGs (Project Groups, not Paying Guests). <br><br>
-Within a few short months of joining Synapse, he proved himself worthy of being elevated to a Business Head. For the better part of the next 10 years, Ravi has diligently led Synapse from the front. Today is the day we celebrate his journey.</p>
+<h2>Some 4/5 years ago, on a sweltering day of the month of August, a bright young engineer stood overwhelmed by grandeur of the Clock Tower. </h2>
```

Comment:
Also, replace this with "Some 4/5 years ago, on a sweltering day of August, a bright young engineer stood overwhelmed by the grandeur of the Clock Tower."